EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from typing import Callable, Optional
+from time import time as _time
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+
+
+class TimeParser:
+    @staticmethod
+    def get_time_since_epoch(
+        year: int, month: int, day: int, hour: int, minute: int, seconds: float
```

Comment:
Why dont you make hour, minute and seconds optional ?

Reply:
oh